src/javascript/app_2/App/Components/Elements/Tooltip2/index.js

```diff
@@ -0,0 +1 @@
+export Tooltip2 from './tooltip-2.jsx';
```

Comment:
maybe can change the name to `TooltipPose`

Reply:
Why?

Reply:
I think `Popover` is a better choice. We've used the name before for a similar behaving component (which was previously removed from the project).

Reply:
It is all popover now